apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance.jsx

```diff
@@ -194,6 +196,12 @@ export class SessionAttendance extends React.Component {
                   )}
                   <th>Verified Teacher Account</th>
                   {this.showPuzzlesCompleted && <th>Puzzles Completed</th>}
+                  {this.props.scholarshipWorkshop && (
+                    <th>Code.org Scholarship?</th>
+                  )}
+                  {this.props.scholarshipWorkshop && (
```

Comment:
Can we just combine these two?

Reply:
actually - we can't combine them unless we wrap them in a parent element. since they are table cells, i don't think we want to do that.

Reply:
Oh that's a very interesting scenario.  Fair enough.